tests/workflows/test_core.py

```diff
@@ -92,7 +94,8 @@ async def test_manual_flow(
         assert workflow.transmission_attributes["tenant_id"] == "MyTenant"
         assert "content-length" in workflow.transmission_attributes
         await workflow.synchronize()
-        assert nats_mock.publish.call_count == 5
+        assert nats_mock.publish.call_count == 0
+        assert jetstream_mock.publish.call_count == 5
 
```

Comment:
It's a good idea to leave the check for nats publishes in place, so we catch any use of nats client publishes instead of jetstream.